lib/profile_node.stp

```diff
@@ -443,3 +443,17 @@ probe timer.profile {
         printf("\n");
 	}
 }
+
+probe timer.profile {
+    /*
+     * Since we run completely asynchronously with respect to
+     * V8's activities, we can actually catch the heap in temporarily
+     * inconsistent states, causing bad copyins and such.  Try-catch
+     * so we drop a data point in that case rather than halting.
+     */
+    try {
+        take_node_backtrace()
+    } catch {
+        next
```

Comment:
That is a good observation!  The answer is yes, the way the code works today.  For performance and sanity, I'm planning to make this code start building strings instead of printing inline, but today you can get garbled stacks.  I think I can't address that without quite a bit of refactoring (which I fully intend us to do).